browser/src/control/Control.PartsPreview.js

```diff
@@ -480,7 +480,7 @@ window.L.Control.PartsPreview = window.L.Control.extend({
 
 	_setPart: function (e) {
 		if (cool.Comment.isAnyEdit()) {
-			cool.CommentSection.showCommentEditingWarning();
+			app.sectionContainer.getSectionWithName(app.CSections.CommentList.name).navigateAndFocusComment(cool.Comment.isAnyEdit());
```

Comment:
you need to cast to the concrete section type `CommentListSection` then we can use method which is declared only in that class - if that was the reason to move declaration of `navigateAndFocusComment` into generic base class